open-metadata-test/open-metadata-fvt/open-types-fvt/open-types-test/src/main/java/org/odpi/openmetadata/fvt/opentypes/entities/InformationView/InformationViewReferences.java

```diff
@@ -0,0 +1,989 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+// This is a generated file - do not edit - changes should be made to the templates amd/or generator to generate this file with changes.
```

Comment:
It says this is generated file.
Does it means suppose NOT to be committed.

Reply:
Even more importantly... how is this file now in the proposed commit given that it was deleted a while ago?
Have any force-push been used (which could contaminate history?)
Could this cause any other odd changes to be pulled in?

Reply:
I can see the commit #51c9ab4e9c180cf6226250c1b8f07d76ba990176 is also in this branch, so it's
likely this file was accidentally re-added, before the later fix to remove from .gitignore

Please remove from the commit.

Reply:
No force push and will not be used without agreement